build/verify-archive.sh

```diff
@@ -9,7 +9,7 @@
 set -euo pipefail
 
 apt-get update
-apt-get install -y autoconf bison cmake libncurses-dev
+apt-get install -y autoconf bison cmake libncurses-dev nodejs
```

Comment:
Yeah, this appears to be necessary. still not sure how it worked before.

Also, not sure how we'd get it to install 10, instead of the latest 🤔 

Sooner or later, we should upgrade our stuff so it can build on latest node…